### PR TITLE
Small PR fixing dart format on android_alarm_manager_plus

### PR DESCRIPTION
**Pull request by miquelbeltran**

### CODE REVIEW

File: packages/android_alarm_manager_plus/lib/android_alarm_manager_plus.dart [hunk 1]
```diff
@@ -78,8 +78,10 @@ class AndroidAlarmManager {
   /// This is exposed for the unit tests. It should not be accessed by users of
   /// the plugin.
   @visibleForTesting
-  static void setTestOverides(
-      {_Now? now, _GetCallbackHandle? getCallbackHandle}) {
+  static void setTestOverides({
```
Comment (jpnurmi) [inline packages/android_alarm_manager_plus/lib/android_alarm_manager_plus.dart:81]: nit-pick: there's a typo "overides"

**Review by jpnurmi — DISMISSED** [review]
perhaps we could run melos format for the entire code base while at it? i noticed there were quite a few changes all over when i run it last time

**Comment by miquelbeltran** [comment]
> run melos format for the entire code base while at it?

we are already doing that through CI if I am not wrong. This formatting issue was not detected with it for example.

**Review by jpnurmi — APPROVED** [review]

**Comment by jpnurmi** [comment]
For some reason, I get a lot of changes when I run melos format on Linux. Perhaps a different clang-format version or something like that? 90% of it seems autogenerated examples, though.

```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   packages/battery_plus/example/linux/flutter/generated_plugin_registrant.cc
	modified:   packages/battery_plus/example/linux/main.cc
	modified:   packages/battery_plus/example/linux/my_application.cc
	modified:   packages/battery_plus/example/windows/runner/resource.h
	modified:   packages/battery_plus/example/windows/runner/run_loop.h
	modified:   packages/battery_plus/example/windows/runner/win32_window.cpp
	modified:   packages/battery_plus/example/windows/runner/win32_window.h
	modified:   packages/battery_plus_windows/windows/battery_plus_windows_plugin.cpp
	modified:   packages/battery_plus_windows/windows/include/battery_plus_windows/system_battery.h
	modified:   packages/battery_plus_windows/windows/system_battery.cpp
	modified:   packages/connectivity_plus/example/linux/flutter/generated_plugin_registrant.cc
	modified:   packages/connectivity_plus/example/linux/main.cc
	modified:   packages/connectivity_plus/example/linux/my_application.cc
	modified:   packages/connectivity_plus/example/windows/flutter/generated_plugin_registrant.cc
	modified:   packages/connectivity_plus/example/windows/runner/main.cpp
	modified:   packages/connectivity_plus/example/windows/runner/resource.h
	modified:   packages/connectivity_plus/example/windows/runner/run_loop.h
	modified:   packages/connectivity_plus/example/windows/runner/utils.cpp
	modified:   packages/connectivity_plus/example/windows/runner/win32_window.cpp
	modified:   packages/connectivity_plus/example/windows/runner/win32_window.h
	modified:   packages/network_info_plus/example/windows/runner/main.cpp
	modified:   packages/network_info_plus/example/windows/runner/resource.h
	modified:   packages/network_info_plus/example/windows/runner/run_loop.h
	modified:   packages/network_info_plus/example/windows/runner/utils.cpp
	modified:   packages/network_info_plus/example/windows/runner/win32_window.cpp
	modified:   packages/network_info_plus/example/windows/runner/win32_window.h
	modified:   packages/network_info_plus_windows/windows/include/network_info_plus_windows/network_info.h
	modified:   packages/network_info_plus_windows/windows/network_info.cpp
	modified:   packages/network_info_plus_windows/windows/network_info_plus_windows_plugin.cpp
	modified:   packages/package_info_plus/example/linux/flutter/generated_plugin_registrant.cc
	modified:   packages/package_info_plus/example/linux/my_application.cc
	modified:   packages/package_info_plus/example/windows/flutter/generated_plugin_registrant.cc
	modified:   packages/package_info_plus/example/windows/runner/resource.h
	modified:   packages/package_info_plus/example/windows/runner/run_loop.h
	modified:   packages/package_info_plus/example/windows/runner/win32_window.cpp
	modified:   packages/package_info_plus/example/windows/runner/win32_window.h

no changes added to commit (use "git add" and/or "git commit -a")
```

**Comment by miquelbeltran** [comment]
right, same for me, seems related to clang-format

**Pull request merged**